packages/byods/src/BaseClient.ts

```diff
@@ -0,0 +1,184 @@
+import fetch, {Response, RequestInit} from 'node-fetch';
+import TokenManager from './TokenManager';
```

Comment:
nitpik- Let's leave a line between local import and node_module import.

Reply:
Done